Device Agent UI (#92)
FlowFuse/device-agent: README.md

## Running as a service

An example service file is provided [here](https://github.com/flowforge/flowforge-device-agent/tree/main/service).

## Running the agent with the Web UI enabled

> The agent can be run with a Web UI interface. This is an optional feature and
> is not enabled by default.  The UI permits the user to view the current status of 
> the agent and the Node-RED instances it is running.  It also permits
> the user to download a device configuration or a provisioning configuration file.

Reviewer: ```suggestion
The agent can be run with a Web UI interface. This is an optional feature and
is not enabled by default.  The UI permits the user to view the current status of 
the agent and the Node-RED instances it is running.  It also permits
the user to download a device configuration or a provisioning configuration file.
```

I think the emphasis needs to be about this being a UI for doing initial configuration/setup of the agent - it isn't a general purpose Web UI for monitoring the status of the agent, especially given it shuts down after a short period of time.

> ## Running the agent with the Web UI enabled
> 
> The agent can be run with a Web UI interface. This is an optional feature and
> is not enabled by default.  The UI permits the user to view the current status of 
> the agent and the Node-RED instances it is running.  It also permits
> the user to download a device configuration or a provisioning configuration file.
> 
> To enable the UI, use the `-w/--ui` option. This will start a web server on
> the specified host and port. The default host is `0.0.0.0` and the default port is `1879`.

Author: ```suggestion
## Running the agent with the Web UI enabled

The Device Agents Web UI is provided to enable the user to download a device
configuration or a provisioning configuration file. This is an optional feature and
is not enabled by default.

To enable the UI, use the `-w/--ui` option. This will start a web server on
the specified host and port. The default host is `0.0.0.0` and the default port is `1879`.
```

Author: To minimise friction on review, I will merge this change on emphasis so that the reviewer is not looking in two places.
